dra-evolution/pkg/api/capacity_types.go

```diff
@@ -126,6 +126,10 @@ type DeviceResourceValue struct {
 	IntRangeValue *IntRange `json:"intRange,omitempty" protobuf:"varint,2,rep,name=intRange"`
 }
 
+// Format TBD. We should not repeat the mistake made with resource.Quantity
+// and define a type that changes during round-tripping.
+type IntRange string
+
```

Comment:
Why is this needed? Did I forget to add my file `intrange.go` that defines this type?

Reply:
Pushed